site/content/en/docs/Tutorials/Recommendation/recommendation-framework.md

```diff
@@ -166,10 +166,17 @@ data:
             apiVersion: apps/v1
           - kind: StatefulSet
             apiVersion: apps/v1
+        config:                              # settings for recommender
```

Comment:
No need to add here, better to update here :https://github.com/gocrane/crane/blob/main/site/content/en/docs/Tutorials/Recommendation/resource-recommendation.md
configuration part